fastchat/model/model_adapter.py

```diff
@@ -1202,6 +1201,7 @@ def get_default_conv_template(self, model_path: str) -> Conversation:
 register_model_adapter(PythiaAdapter)
 register_model_adapter(InternLMChatAdapter)
 register_model_adapter(StarChatAdapter)
+register_model_adapter(PeftModelAdapter)
```

Comment:
This change will fundamentally break loading any peft based model that includes a name that matches any of the above core model adapters (such as `fastchat-my-fancy-peft`).  What problem are you trying to resolve by moving it down below?  The latest changes to checking for peft models checks the actual model config rather than the name.  Are you finding that the peft model is being loaded when it shouldn't?

Reply:
I think they try to avoid this network request https://github.com/lm-sys/FastChat/blob/7102dba7d9f7e9145b40352afd0ce74f3219b6e2/fastchat/model/model_adapter.py#L393